Alignment/HIPAlignmentAlgorithm/plots/HIPplots_cc.d

```diff
@@ -0,0 +1,109 @@
+./HIPplots_cc.so: /afs/cern.ch/sw/lcg/app/releases/ROOT/5.34.09/x86_64-slc6-gcc47-dbg/root/include/TProfile.h
+./HIPplots_cc.so: /afs/cern.ch/sw/lcg/app/releases/ROOT/5.34.09/x86_64-slc6-gcc47-dbg/root/include/TPaveStats.h
+./HIPplots_cc.so: /afs/cern.ch/sw/lcg/app/releases/ROOT/5.34.09/x86_64-slc6-gcc47-dbg/root/include/TPaveText.h
+./HIPplots_cc.so: /afs/cern.ch/sw/lcg/app/releases/ROOT/5.34.09/x86_64-slc6-gcc47-dbg/root/include/TPave.h
+./HIPplots_cc.so: /afs/cern.ch/sw/lcg/app/releases/ROOT/5.34.09/x86_64-slc6-gcc47-dbg/root/include/TBox.h
+./HIPplots_cc.so: /afs/cern.ch/sw/lcg/app/releases/ROOT/5.34.09/x86_64-slc6-gcc47-dbg/root/include/TText.h
+./HIPplots_cc.so: /afs/cern.ch/sw/lcg/app/releases/ROOT/5.34.09/x86_64-slc6-gcc47-dbg/root/include/TLine.h
+./HIPplots_cc.so: /afs/cern.ch/sw/lcg/app/releases/ROOT/5.34.09/x86_64-slc6-gcc47-dbg/root/include/TNtuple.h
+./HIPplots_cc.so: /afs/cern.ch/sw/lcg/app/releases/ROOT/5.34.09/x86_64-slc6-gcc47-dbg/root/include/TMatrixD.h
+./HIPplots_cc.so: /afs/cern.ch/sw/lcg/app/releases/ROOT/5.34.09/x86_64-slc6-gcc47-dbg/root/include/TMatrixT.h
+./HIPplots_cc.so: /afs/cern.ch/sw/lcg/app/releases/ROOT/5.34.09/x86_64-slc6-gcc47-dbg/root/include/TMatrixTBase.h
+./HIPplots_cc.so: /afs/cern.ch/sw/lcg/app/releases/ROOT/5.34.09/x86_64-slc6-gcc47-dbg/root/include/TMatrixTUtils.h
+./HIPplots_cc.so: /afs/cern.ch/sw/lcg/app/releases/ROOT/5.34.09/x86_64-slc6-gcc47-dbg/root/include/TMatrixDfwd.h
+./HIPplots_cc.so: /afs/cern.ch/sw/lcg/app/releases/ROOT/5.34.09/x86_64-slc6-gcc47-dbg/root/include/TMatrixDUtils.h
+./HIPplots_cc.so: /afs/cern.ch/sw/lcg/app/releases/ROOT/5.34.09/x86_64-slc6-gcc47-dbg/root/include/TMatrixDUtilsfwd.h
+./HIPplots_cc.so: /afs/cern.ch/sw/lcg/app/releases/ROOT/5.34.09/x86_64-slc6-gcc47-dbg/root/include/TVectorD.h
+./HIPplots_cc.so: /afs/cern.ch/sw/lcg/app/releases/ROOT/5.34.09/x86_64-slc6-gcc47-dbg/root/include/TVectorT.h
+./HIPplots_cc.so: /afs/cern.ch/sw/lcg/app/releases/ROOT/5.34.09/x86_64-slc6-gcc47-dbg/root/include/TMatrixTSym.h
+./HIPplots_cc.so: /afs/cern.ch/sw/lcg/app/releases/ROOT/5.34.09/x86_64-slc6-gcc47-dbg/root/include/TMatrixTSparse.h
+./HIPplots_cc.so: /afs/cern.ch/sw/lcg/app/releases/ROOT/5.34.09/x86_64-slc6-gcc47-dbg/root/include/TMatrixDSym.h
+./HIPplots_cc.so: /afs/cern.ch/sw/lcg/app/releases/ROOT/5.34.09/x86_64-slc6-gcc47-dbg/root/include/TMatrixDSymfwd.h
+./HIPplots_cc.so: /afs/cern.ch/sw/lcg/app/releases/ROOT/5.34.09/x86_64-slc6-gcc47-dbg/root/include/TMatrixDSparse.h
+./HIPplots_cc.so: /afs/cern.ch/sw/lcg/app/releases/ROOT/5.34.09/x86_64-slc6-gcc47-dbg/root/include/TMatrixDSparsefwd.h
+./HIPplots_cc.so: /afs/cern.ch/sw/lcg/app/releases/ROOT/5.34.09/x86_64-slc6-gcc47-dbg/root/include/TLegend.h
+./HIPplots_cc.so: /afs/cern.ch/sw/lcg/app/releases/ROOT/5.34.09/x86_64-slc6-gcc47-dbg/root/include/cintdictversion.h /afs/cern.ch/sw/lcg/app/releases/ROOT/5.34.09/x86_64-slc6-gcc47-dbg/root/include/RVersion.h
+HIPplots_cc__ROOTBUILDVERSION= 5.34/09
```

Comment:
The same applies to the `HIPplots_cc.so` and `HIPplots_cc_ACLiC_dict_rdict.pcm`.